### ISSUE TITLE
Invalid header?

### ISSUE THREAD

**Issue by tkossak** [rianhunter/dbxfs#19]
When I try to store access token in gpg file, using config entry:

```
"access_token_command": ["gpg2", "--decrypt", "/mnt/params/.password-store/mine/dropbox_access_token.gpg"]
```

I get the error:

```
Running 'gpg2 --decrypt /mnt/params/.password-store/mine/dropbox_access_token.gpg' for access token
gpg: encrypted with 2048-bit RSA key, ID <ID_HERE>, created 2015-09-12
      "Kossak_org <MAIL>"
Error using access token: Invalid header value b'Bearer <MY_ACCESS_TOKEN_HERE>\n'
Want to try entering the access token directly? [y/N] 
```

What do I do wrong? When I paste the same access token interactively it works as it should.

**Comment by rianhunter** [comment]
Looks like that GPG file contains "<MY_ACCESS_TOKEN_HERE>" which is not a valid access token.

**Comment by rianhunter** [comment]
Oh I see that's an edit you made. The problem is whitespace. Make sure the plaintext access token encoded in the GPG file is not surrounded by whitespace.

**Comment by tkossak** [comment]
I noticed new line in dbxfs output but I thought it's not in the gpg file directly. It turns out it was in the gpg file, because my vim automatically adds `\n` at the end of every text file and doesn't display it when I open the file in text mode!
I fixed it by running `pass edit /mnt/params/.password-store/mine/dropbox_access_token.gpg` (I use [pass](https://www.passwordstore.org/) to manage secrets) and running in vim:
```
:set binary
:set noeol
:wq
```

and it works now.

Thank you.
